### PR TITLE
Makes "on station" a bit more literal and inclusive for scoreboards

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[tweak][bugfix]

## What this does
if theres messes or trash in space near the station like from meaty gores, it no longer counts as "on station" now

## Why it's good
more literal and stops people having to spacewalk to do janitor work

## Changelog
:cl:
 * tweak: Messes and trash found outside of the station on the same z-level no longer count as "on station" for scoreboards
 * tweak: If nothing shows up under "the bad" in the scoreboard, a special message is given relating to it
 * bugfix: Station power, atmos alarms, messes and trash no longer only count for the main level on multi-Z maps

### CODE REVIEW

File: code/game/gamemodes/endgame/scoreboard/department_score.dm [hunk 1]
```diff
@@ -76,19 +76,16 @@
 
 /datum/controller/gameticker/scoreboard/proc/service_score()
 	//Janitor
-	//Check how many uncleaned mess are on the station. We can't run through cleanable for reasons, so yeah, long
+	//Check how many uncleaned mess are on the station.
+	var/list/valid_areas = the_station_areas.Copy() - typesof(/area/holodeck) //excluding these for now because subtypes are on z2 normally
```
Comment (west3436) [inline code/game/gamemodes/endgame/scoreboard/department_score.dm]: Does this mean walking into the holodeck counts as being off-station?

File: code/game/objects/effects/decals/cleanable.dm [hunk 2]
```diff
@@ -183,12 +183,6 @@ var/list/infected_cleanables = list()
 			perp.add_blood_to_feet(amount, basecolor, blood_DNA)
 			amount--
 
-/obj/effect/decal/cleanable/proc/messcheck(var/obj/effect/decal/cleanable/M)
```
Comment (west3436) [inline code/game/objects/effects/decals/cleanable.dm:186]: Why is this here??

Comment (SECBATON-GRIFFON) [reply code/game/objects/effects/decals/cleanable.dm:186]: not really sure, probably legacy code

File: code/game/gamemodes/endgame/scoreboard/department_score.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/game/gamemodes/endgame/scoreboard/department_score.dm]: not anymore, fixed. also, the holodeck always counted as being on station, this is just for mess checks

**Comment by jwhitak** [comment]
I want to disagree with this PR if only to force jannies to go on spacewalks to clean up the exterior of the station too. It's gotta look nice for the NT reps when they fly in...

File: code/game/objects/effects/decals/cleanable.dm (hunk 2 above)
Comment (west3436) [reply code/game/objects/effects/decals/cleanable.dm:186]: No I mean why are you including this change in the PR? Don't remove things that aren't in-scope

Comment (SECBATON-GRIFFON) [reply code/game/objects/effects/decals/cleanable.dm:186]: proc becomes unused after this, wasn't even doing anything anyways
